### ISSUE TITLE
Avoid using SafariViewController when token expires

### ISSUE THREAD

**Issue by mmoosbac94** [p2/OAuth2#418]
Hello everybody,

we currently have the issue that we have to avoid the use of safari services. Our goal is to use the os standard browser for authentication. Its working when we first authorize via "oauth2.authorize()". But if the token expires, the SafarViewController is shown for new authentication. Is there a way to not use the SafariViewController and use the os standard browser instead for a new authentication as well?

Thank you

**Comment by ossus-lib** [comment]
Even if `authorizeEmbedded` is set to false?

**Comment by mmoosbac94** [comment]
Yes, and when we start the app its working. The standard default browser will be opened. But if the token expires and there is no refresh token and I am in app, then the Safari Viewcontroller is shown instead of the browser. We already tried different advanced settings from documentation but without success.

**Comment by ossus-lib** [comment]
Can you reproduce with a debug logger and paste the logs here?

```
oauth2.logger = OAuth2DebugLogger(.trace)
```

**Comment by mmoosbac94** [comment]
We tried to get some logs, but the authorization function is not called again when token expires. So when token expires there are no visible p2/oauth2 specific logs. We are using a so called ArcGIS SDK. We are not sure if this SDK is responsible for showing the SafariViewController.
